changelog/feature-deposits-payouts-rename

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.
```

Comment:
To make this changelog file not to create a changelog entry:

```suggestion
Significance: patch
Type: dev
Comment: Rename 'deposit' to 'payout' across various places in the WooPayments admin UI.

```

Reply:
This is the main changelog for the PR and should be probably kept as it is?